index.html

```diff
@@ -87,44 +74,34 @@
           <h3><a class="fn org url" href="http://www.meetup.com/NottAgile/">Agile Nottingham</a></h3>
           <p class="who">Agile in Nottingham</p>
           <p class="when">Second Last Wednesday</p>
-        </li>
+        <span><i class="fa fa-calendar"></i> <a href="http://www.meetup.com/NottAgile/events/232778488/">Wednesday 19th October 2016 at 6:30pm</a> </span></li>
         <li class="bcs vcard">
-          <h3><a class="fn org url" href="http://www.meetup.com/Women-In-Tech-Nottingham">Women In Tech Nottingham </a></h3>
+           <h3><a class="fn org url" href="http://www.meetup.com/Women-In-Tech-Nottingham">Women In Tech Nottingham </a></h3>
           <p class="who">Women In Tech Nottingham</p>
           <p class="when">First Thursday</p>
+        <span><i class="fa fa-calendar"></i> <a href="http://www.meetup.com/Women-In-Tech-Nottingham/events/232371368/">Thursday 6th October 2016 at 7:00pm</a> </span></li>
+         <li class="bcs vcard">
+          <h3><a class="fn org url" href="http://www.healthtech.org.uk">HealthTech Nottingham </a></h3>
+          <p class="who">HealthTech Nottingham</p>
+          <p class="when">First Thursday of the month</p>
         </li>
         <li class="bcs vcard">
           <h3><a class="fn org url" href="http://www.meetup.com/NottinghamR-Nottingham-R-Users-Group/">Nottingham R User Group</a></h3>
           <p class="who">R Users in  Nottingham</p>
           <p class="when">Quarterly</p>
         </li>
-			</ol>
-		</section>
 
-    <section>
-      <header><h2>Twitter</h2></header>
-        <a class="twitter-timeline" href="https://twitter.com/dxnevent/lists/nottingham-events" data-widget-id="700654764040654851">Tweets from https://twitter.com/dxnevent/lists/nottingham-events</a> <script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+"://platform.twitter.com/widgets.js";fjs.parentNode.insertBefore(js,fjs);}}(document,"script","twitter-wjs");</script>
+      </ol>
     </section>
+  <section>
+      <header id="twitter-feed"><h2>Twitter</h2></header>
+      <div class="twitter-feed">
+        <a class="twitter-timeline" href="https://twitter.com/dxnevent/lists/nottingham-events" data-widget-id="700654764040654851">Tweets from https://twitter.com/dxnevent/lists/nottingham-events</a> <script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+"://platform.twitter.com/widgets.js";fjs.parentNode.insertBefore(js,fjs);}}(document,"script","twitter-wjs");</script>
+    </div> 
+   </section>
 
-	</article>
-
-	<footer>
-		<p>A <a href="http://sneeu.com/">sneeu.com</a> thing.</p>
-	</footer>
-
-    <script src="js/jquery.js"></script>
-    <script src="js/event.js"></script>
-<script>
-  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
```

Comment:
Can the google analytics be left in? Quite handy for seeing whether anyone is using the site.